### DISCUSSION TITLE
Important Mistakes in Musescore 4

### DISCUSSION THREAD

**Discussion by Antonio2007Dev** [musescore/MuseScore#13315] [General]
I have tested the lastest nightly version, and I have noticed that there are some playback problems in the scores.
1.The reproduction of the scores is not so smooth, in some parts it sounds a bit robotic and choppy (well I noticed that because I have a very sensitive ear).
2. There are some errors such as: a staccato is heard when there is not, some extensions with slurs are not heard perfectly (the duration of the note is cut), grace notes are not played correctly (sometimes they are heard as staccato , others are not even heard, and sometimes the duration of them is not correct).
3. When playing the score with dynamic "p" and setting the pedal specification, the notes don't lengthen (sounds like I'm not using the pedal) and also sounds like it's with dynamic "f".
4. Also sometimes notes are played as if they had an accent aboce.

In terms of performance and interface, I think that M4 is very good, but I consider the reproduction of the scores to be a very important function that must be polished and fixed perfectly, I hope that no other problem will stop me, but if it is So I hope someone else notices it.

**Comment by MarcSabatella** [comment]
Yes, most of these are known problems that have been reported earlier. But in order to be sure, could you please post a link to one such score and describe for each of these separate issues, which measure you feel demonstrates it?

I think all but the second part of #3 have issues submitted.  That’s one I had noticed and was trying to figure out how to reproduce today.
